Start.ps1

```diff
@@ -46,7 +46,7 @@ function Setup
 #Builds the Stuff
 function BuildSwf
 {
-	(Get-Content $xml) -replace '<versionNumber>(.*)</versionNumber>', ('<versionNumber>'+((${latestVersion}) -split "_")[-1]+'</versionNumber>')| Set-Content $xml
+	[Regex]::Replace( $(Get-Content $xml), '(?s)/<versionNumber>.*?</versionNumber>/', ('<versionNumber>'+((${latestVersion}) -split "_")[-1]+'</versionNumber>') ) | Set-Content $xml

```

Comment:
I believe it may be time to split this line with these changes.

Reply:
I won't do much changes here, I just wanted to fix the bug the most cleanly possible, that's my first powershell script fix ;)